docs/UserGuide.md

```diff
@@ -105,8 +105,33 @@ Format: `deletelog n/[NAME] id/[LOG_INDEX] -a`
 * If no `NAME` or `LOG_INDEX` is provided, but `-a` is provided, then all possible logs
   of all friends will be deleted.
 
+### Show all friends : `showfriends`
+
+Shows all friends in the address book. Switches GUI to the friends tab.
+
+Format: `showfriends`
+
+### Show a specific friend: `showfriend`
+
+Shows page containing the full details related to an existing friend in the address book. By default, only the latest 10 logs are displayed in this page.
+
+Format: `showfriend n/NAME -alllogs`
+
+* NAME field **must** be provided.
+* If the `-alllogs` flag is provided all the logs contained in that person will be displayed instead of the default 10 logs.
+
+Examples:
+* `showfriend n/John Doe` Will open up the page containing full details related to John Doe - his name, address, phone number, email, description, logs, and upcoming events with him.
```

Comment:
I like how you described what the examples will do! We should probably standardize this across the board though. I was thinking that maybe this should be done only for the more complicated/less intuitive commands?